### PR TITLE
Lps 125421 7 8

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-125421

/cc @dantewang @tinatian

### CODE REVIEW

**Comment by vicnate5** [comment]
Majority of the tests failed to run here.

Retrying

**Comment by vicnate5** [comment]
ci:test:relevant

**Comment by vicnate5** [comment]
ci:test:acceptance

**Comment by slnn** [comment]
Rebase with master and rerun the ci test

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:acceptance

**Comment by dantewang** [comment]
@slnn Some of your commits, like https://github.com/liferay-core-infra/liferay-portal/pull/1067/commits/6318a8d472560b75c1b3741b61bd9254ef79e0a8 and https://github.com/liferay-core-infra/liferay-portal/pull/1067/commits/b3a30eddaa17eb112783afc1ea387debdf575c87, should be with message "SF, xxx" to let people know you don't change any logic.

File: benchmarks/build.xml [hunk 1]
```diff
@@ -32,8 +32,8 @@
 	<target name="clean">
 		<delete includeemptydirs="true">
 			<fileset
-				dir="."
-				includes="*.sql,benchmarks-actual.properties, liferay/**, ${build.dir}/**"
+				dir="${basedir}"
+				includes="benchmarks-actual.properties, liferay/**, ${build.dir}/**"
```
Comment (dantewang) [inline benchmarks/build.xml:51]: @slnn The commit is "Do cleanup before each invoke" but actually you also changed the basedir and removed an entry from the fileset. Please split commits to make it clear, or drop the changes if they are not actually needed.

**Comment by slnn** [comment]
Hi @dantewang 

Follow your comments, I have double-checked and updated this pull.

Lily

**Comment by catarineaguiar** [comment]
ci:test:acceptance

File: benchmarks/build.xml [hunk 2]
```diff
@@ -3,10 +3,14 @@
 <project basedir="." default="run" name="benchmarks">
 	<import file="../build-common.xml" />
 
-	<target name="build-sample-sql">
+	<property name="build.dir" value="build" />
+
+	<target depends="clean" name="build-sample-sql">
+		<property name="sample.sql.output.dir" value="${basedir}/${build.dir}" />
```
Comment (tinatian) [inline benchmarks/build.xml:9]: @slnn Please remove this, make benchmark project copy the output to its target dir.

File: modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java [hunk 3]
```diff
@@ -61,13 +68,13 @@ public SampleSQLBuilder() {
 
 		tempDir.mkdirs();
 
-		Reader reader = generateSQL();
+		Reader reader = mergeSQLTemplates();
```
Comment (tinatian) [inline modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/SampleSQLBuilder.java:71]: @slnn Please clean up the method name and its scope.

**Comment by tinatian** [comment]
@slnn Please see my comments and resend.

Thanks.
Tina.

**Pull request closed without merging**